bgpd/bgp_vty.c

```diff
@@ -6104,7 +6104,8 @@ DEFUN_NOSH (address_family_ipv4_safi,
 		safi_t safi = bgp_vty_safi_from_str(argv[2]->text);
 		if (bgp->inst_type != BGP_INSTANCE_TYPE_DEFAULT &&
 		    safi != SAFI_UNICAST && safi != SAFI_MULTICAST) {
-			vty_out(vty, "Only Unicast and Multicast SAFIs supported in non-core instances.\n");
```

Comment:
should not you ad && safi != SAFI_EVPN, if you really want to unblock ?

Reply:
sure, will make the change to make it consistent to the address family l2vpn command. This command doesn't really take EVPN safi so we are good without the check as well.